src/ClusterBootstrap/cloud_init_deploy.py

```diff
@@ -557,13 +556,26 @@ def render_worker_node_specific(config, args):
                               "./deploy/cloud-config/cloud_init_worker_{}.txt".format(sku), config)
 
 
+def render_elasticsearch_node_specific(config, args):
+    if not "elasticsearch_node" in config or len(config["elasticsearch_node"]) == 0:
+        print("Warning: no elasticsearch node specified, logging service might not work as expected.")
+        return
+    config.pop("mount_and_link", [])
+    config = escaped_etcd_end_point_and_k8s_api_server(config)
+    example_node_name = config["elasticsearch_node"][0].split(".")[0]
```

Comment:
Naming `example_node_name` is not clear to me at first glance.

Reply:
to extrapolate to other roles as well. can discuss this together in the separate PR mentioned above.

Reply:
`first_node`? Since you are literally just taking the first element in the node list.

Reply:
ok, I'll rename it to first_node in that refactor PR.